Source/Details/_ASCollectionViewCell.mm

```diff
@@ -94,6 +95,29 @@ - (void)layoutSubviews
   self.node.frame = self.contentView.bounds;
 }
 
+- (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
+{
+  /**
+   * The documentation for hitTest:withEvent: on an UIView explicitly states the fact that:
+   * it ignores view objects that are hidden, that have disabled user interactions, or have an
+   * alpha level less than 0.01.
+   * To be able to determine if the collection view cell should skip going further down the tree
+   * based on the states above we use a valid point within the cells bounds and check the
+   * superclass hitTest:withEvent: implementation. If this returns a valid value we can go on with
+   * checking the node as it's expected to not be in one of these states.
+   */
+  if (![super hitTest:[self convertPoint:self.center fromView:self.superview] withEvent:event]) {
```

Comment:
Rather than converting the point from superview just to be simpler we should use `self.bounds.origin`